quill/common/src/bundle.rs

```diff
@@ -0,0 +1,45 @@
+use crate::Component;
+
+pub trait EntityBuilder {
```

Comment:
Should maybe be named something like `BundleSpawner` or `BundleAdder` or maybe straight up make the entity builder a trait.